lib/utils.js

```diff
@@ -142,3 +142,11 @@ exports.runManualTestCase = async (test, testCase) => {
   printInstructionsSet('CLEANUP', testCase.cleanup)
   test.true(await ipc.requestConfirmation())
 }
+
+exports.getDeviceUptime = (ssh) => {
```

Comment:
I think this should be part of the "resinos" component, and then you can get rid of the `ssh` argument

Reply:
I agree that this can be in resinos, but the ssh argument is still required as we connect via the `resin-proxy`, hence we would need resinio ssh config.

Reply:
Ah, I see what you mean. Looks fine then.